### DISCUSSION TITLE
Return `_scd` table for connections

### DISCUSSION THREAD

**Discussion by alandmorford** [airbytehq/airbyte#31681] [Connector Ideas and Features]
With Destination V2, it seems the _SCD tables were found to be generally not useful.
These tables were a primary motivator to use AirByte, I can understand not having them being a default behavior if not generally useful but dropping support altogether is surprising.

**Comment by marcosmarxm** [comment]
Thanks for the feedback. I'll keep it open to receive more comments and discussion from other users.
The current roadmap for destination doesn't include the history table.

**Comment by evantahler** [comment]
Linking this discussion back to https://github.com/airbytehq/airbyte/issues/25194.

The reasons we removed SCD tables were:
1. They were inaccurate.  
    * Except in the case of a CDC sync which never had a reset, Airbyte's old implementations of SCD tables only sampled data when a sync happened, missing intermediate changes between syncs.  For the majority of sources, CDC (or something like it) is not available. 
3. They were rarely used
    * The vast majority of Airbyte users only used the final normalized tables.
5. Their creation added cost and slowed down syncs
    * The cost of normalization in your warehouse was (at least) doubled by the creation of SCD tables in addition to the final tables

All of this said, you can recreate similar SCD tables by running dbt at the end of a sync and creating 'snapshot' tables (https://docs.getdbt.com/docs/build/snapshots).

[a comment by an automated account is not shown]
